seq2science/rules/get_fastq.smk

```diff
@@ -176,17 +183,24 @@ rule sra2fastq_PE:
         tmpdir={config[sra_dir]}/tmp/{wildcards.sample}
         mkdir -p $tmpdir; trap "rm -rf $tmpdir" EXIT
 
-        # dump to tmp dir
-        parallel-fastq-dump -s {input}/* -O $tmpdir {config[split]} \
-        --threads {threads} --gzip >> {log} 2>&1
+        # acquire the lock
+        (
+            flock --timeout 30 200 || exit 1
+            sleep 2
+        ) 200>{layout_cachefile_lock}
+
+        # dump
+        fasterq-dump -s {input}/* -O $tmpdir --threads {threads} --split-3 >> {log} 2>&1
```

Comment:
nitpick: --split-3 seems the default for `fasterq-dump`

Reply:
I know, I just thought this is more clear in what the difference is between those two

Reply:
👍